package-files/preremove.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
+
+set -eu
+
+systemctl_stop_service() {
+  if command -v systemctl >/dev/null 2>&1; then
+    systemctl stop infra
+    systemctl disable infra
```

Comment:
Do we need `|| true` here, or some other way to ignore the error? The service isn't necessarily running or enabled.